docs/tutorials/basic/ndarray.md

```diff
@@ -176,13 +176,13 @@ b = a.reshape((2,3,4))
 b.asnumpy()
 ```
 
-The `concatenate` method stacks multiple arrays along the first axis. (Their
+The `concat` method stacks multiple arrays along the first axis. (Their
```

Comment:
Please remove the parentheses. Generally, a parenthetical clause shouldn't stand as its own sentence.